From 43165d5687be85e389d540d9131d09d5f2e417bf Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Tue, 13 Aug 2024 21:46:31 +0530
Subject: [PATCH 035/128] hifive-premier-p550: Remove 1.5GHz to 1.8GHz cpu freq

Removed 1.5GHz to 1.8GHz cpu freq from opp-table as the max
supported freq for CPU is  1.4GHz

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
---
 arch/riscv/boot/dts/eswin/eic7700.dtsi | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700.dtsi b/arch/riscv/boot/dts/eswin/eic7700.dtsi
index 01be763cdf5e..82b62135ebea 100644
--- a/arch/riscv/boot/dts/eswin/eic7700.dtsi
+++ b/arch/riscv/boot/dts/eswin/eic7700.dtsi
@@ -96,26 +96,6 @@ opp-1400000000 {
 			opp-microvolt = <800000>;
 			clock-latency-ns = <70000>;
 		};
-		opp-1500000000 {
-			opp-hz = /bits/ 64 <CLK_FREQ_1500M>;
-			opp-microvolt = <800000>;
-			clock-latency-ns = <70000>;
-		};
-		opp-1600000000 {
-			opp-hz = /bits/ 64 <CLK_FREQ_1600M>;
-			opp-microvolt = <800000>;
-			clock-latency-ns = <70000>;
-		};
-		opp-1700000000 {
-			opp-hz = /bits/ 64 <CLK_FREQ_1700M>;
-			opp-microvolt = <800000>;
-			clock-latency-ns = <70000>;
-		};
-		opp-1800000000 {
-			opp-hz = /bits/ 64 <CLK_FREQ_1800M>;
-			opp-microvolt = <800000>;
-			clock-latency-ns = <70000>;
-		};
 	};
 };
 
-- 
2.47.0

